mne_qt_browser/_pg_figure.py

```diff
@@ -3444,6 +3444,9 @@ def hscroll(self, step):
             xmax = self.mne.xmax
             xmin = xmax - self.mne.duration
 
+        if self.mne.filter_coefs is not None:
+            self._rerun_precompute()
```

Comment:
This seems like a weird conditional to me. Why does it matter if there are filter coefs or not that you have to recompute just for a scroll event?